source/_components/mercedesme.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "Mercedes me"
+description: "Instructions on how to integrate Mercedes car with Mercedes me into Home Assistant."
+date: 2018-01-06 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mercedesme.png
+ha_category: Hub
+ha_release: 0.61
+ha_iot_class: "Cloud Polling"
+---
+
+
+The `Mercedes me` component offers integration with the [Mercedes me](https://www.mercedes-benz.com/de/mercedes-me/) cloud service and provides presence detection as well as sensors such as doors, tires, windows and service interval.
```

Comment:
Missing comma: `windows and service interval` -> `windows, and service interval`